frontend/src/scenes/project-homepage/newlySeenPersonsLogic.ts

```diff
@@ -0,0 +1,26 @@
+import { kea } from 'kea'
+
+import { PersonType } from '~/types'
+import api from 'lib/api'
+
+import type { newlySeenPersonsLogicType } from './newlySeenPersonsLogicType'
```

Comment:
Hm I see that this is a pretty common logic pattern `recently/newlySeen{recordings/persons/insights/etc...}` in your recent PR's. Consolidating the shared components of these logics into a shared logic (like we do for insightLogic) might reduce boilerplate and let us reuse some of the recency language in the frontend code. Up to you if you want to tackle this refactor in a separate PR later down the road.

Reply:
Good call - just moved it all into the `projectHomepageLogic` + added a test for it there